### PR TITLE
Alert headsigns

### PR DESCRIPTION
**Pull request by hannahpurcell**
**Asana task**:  Adhoc

Added Govt Center and Maverick to headsign list. Use that for reconstructed alerts on boundaries

- [ ] Needs version bump?

### CODE REVIEW

File: lib/screens/v2/widget_instance/reconstructed_alert.ex [hunk 1]
```diff
@@ -90,14 +94,32 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
...
+      informed_stop_ids = Enum.into(informed_entities, MapSet.new(), & &1.stop)
+
+      :screens
+      |> Application.get_env(:reconstructed_alert_headsign_matchers)
+      |> Map.get(stop_id)
+      |> Enum.find_value(nil, fn {informed, not_informed, headsign} ->
+        if alert_region_match?(to_set(informed), to_set(not_informed), informed_stop_ids),
+          do: headsign,
+          else: false
+      end)
```
Comment (cmaddox5) [inline lib/screens/v2/widget_instance/reconstructed_alert.ex:106]: Something to consider is pulling this out into some common file for this widget and DUPs. The code looks the same so would be less overhead to have it in one place. Up to you though.

File: lib/screens/v2/widget_instance/reconstructed_alert.ex [hunk 2]
```diff
@@ -90,14 +94,32 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
...
     else
       @route_directions
       |> Map.get(route_id)
       |> Enum.at(direction_id)
     end
   end
 
+  defp to_set(stop_id) when is_binary(stop_id), do: MapSet.new([stop_id])
+  defp to_set(stop_ids) when is_list(stop_ids), do: MapSet.new(stop_ids)
+  defp to_set(%MapSet{} = already_a_set), do: already_a_set
```
Comment (cmaddox5) [inline lib/screens/v2/widget_instance/reconstructed_alert.ex:116]: I don't think this case is necessary. Passing in a MapSet to `MapSet.new` will return a MapSet. You should be able to combine this and the clause above it by removing the guard.

```suggestion
  defp to_set(stop_ids), do: MapSet.new(stop_ids)
```

File: lib/screens_web/views/v2/audio/reconstructed_alert_view.ex [hunk 3]
```diff
@@ -109,4 +109,9 @@ defmodule ScreensWeb.V2.Audio.ReconstructedAlertView do
       _ -> ""
     end
   end
+
+  defp adjust_remedy(remedy) when remedy === "Seek alternate route",
+    do: "Please seek an alternate route"
```
Comment (cmaddox5) [inline lib/screens_web/views/v2/audio/reconstructed_alert_view.ex:114]: Nothing wrong with this, but can be done without a guard.

```suggestion
  defp adjust_remedy("Seek alternate route"), do: "Please seek an alternate route"
```

**Review by cmaddox5 — APPROVED** [review]
Looks good as is, but left some suggestions. Your call on implementing them.

File: lib/screens/v2/widget_instance/reconstructed_alert.ex (hunk 2 above)
Comment (hannahpurcell) [reply lib/screens/v2/widget_instance/reconstructed_alert.ex]: Nice!

**Pull request merged**